internal/cluster/transition.go

```diff
@@ -106,6 +111,15 @@ func (th *transitionHandler) PostPrepareForInstallation(sw stateswitch.StateSwit
 		return errors.New("PostPrepareForInstallation invalid argument")
 	}
 	extra := append(append(make([]interface{}, 0), "install_started_at", strfmt.DateTime(time.Now()), "installation_preparation_completion_status", ""), resetLogsField...)
+
+	if common.IsSingleNodeCluster(sCluster.cluster) {
```

Comment:
i think we talked to move this part to installing stage? no?

Reply:
I think I missed it from my end. I have updated the code to set in `InstallCluster`